jbpm-persistence/jbpm-persistence-jpa/src/test/java/org/jbpm/persistence/settings/JpaSettingsTest.java

```diff
@@ -25,6 +26,7 @@
 import static org.powermock.api.mockito.PowerMockito.*;
 import static org.junit.Assert.*;
 
+@Ignore
```

Comment:
Does it mean that these tests will be skipped permanently?

Reply:
I would say so, not much value in this test to be honest

Reply:
IMO if the plan is to never enable the tests again, they should be simply deleted.